src/pkg/selectors/groups.go

```diff
@@ -270,21 +290,27 @@ const (
 	GroupsCategoryUnknown groupsCategory = ""
 
 	// types of data in Groups
-	GroupsGroup         groupsCategory = "GroupsGroup"
-	GroupsTODOContainer groupsCategory = "GroupsTODOContainer"
-	GroupsTODOItem      groupsCategory = "GroupsTODOItem"
+	GroupsGroup groupsCategory = "GroupsGroup"
+
+	// sharepoint
+	GroupsLibraryFolder groupsCategory = "GroupsLibraryFolder"
+	GroupsLibraryItem   groupsCategory = "GroupsLibraryItem"
```

Comment:
Is channel scope is also to be added here?

Reply:
I think you reviewed a version with merge conflicts.  There are no selectors changes in this PR.